internal/app/servicequotas/servicequotas.go

```diff
@@ -58,29 +64,44 @@ func (s *serviceQuotaFetcher) GetStatistics() Statistics {
 
 // GetQuota retrieves and returns the AWS quota value for the specified serviceCode and quotaCode
 func (s *serviceQuotaFetcher) getQuota(serviceCode string, quotaCode string) (float64, error) {
+	_, span := tracer.Start(s.ctx, "get-quota")
+	defer span.End()
+
+	span.SetAttributes(attribute.String("com.qonto.prometheus_rds_exporter.aws.quota.service_code", serviceCode), attribute.String("com.qonto.rds_exporter.aws.quota.code", quotaCode))
```

Comment:
Just for my culture, what would be the goal of adding `com.qonto.prometheus_rds_exporter` prefix here?

Should we align other attributes with this?